opentech/apply/activity/messaging.py

```diff
@@ -135,7 +135,7 @@ def recipients(self, message_type, **kwargs):
         return [None]
 
     def extra_kwargs(self, message_type, **kwargs):
-        if message_type in [MESSAGES.OPENED_SEALED, MESSAGES.REVIEWERS_UPDATED]:
+        if message_type in [MESSAGES.OPENED_SEALED, MESSAGES.REVIEWERS_UPDATED, MESSAGES.SCREENING]:
```

Comment:
The following issue is happening on the site.
```
2019-01-15T17:35:19.901570+00:00 app[web.1]: [2019-01-15 17:35:19,897] loader_tags: WARNING - Exception raised while rendering {% include %} for template 'funds/applicationsubmission_admin_detail.html'. Empty string rendered instead.
...
2019-01-15T17:35:19.901678+00:00 app[web.1]: AttributeError: Cannot find 'author' on ScreeningStatus object, 'related_object__author' is an invalid parameter to prefetch_related()
```

We save the related object on the Activity when it is a model this allows us to link to it within the activity objects. You need to either pass `str(old.screening_status)` from https://github.com/OpenTechFund/opentech.fund/blob/57e9efc58c69e358c3183dded4ba3184034c19b3/opentech/apply/funds/views.py#L130

We use the related object to link to objects with pages, screening status doesn't have a page.

Relevant code.
https://github.com/OpenTechFund/opentech.fund/blob/57e9efc58c69e358c3183dded4ba3184034c19b3/opentech/apply/activity/models.py#L86-L89

https://github.com/OpenTechFund/opentech.fund/blob/57e9efc58c69e358c3183dded4ba3184034c19b3/opentech/apply/activity/templates/activity/include/listing_base.html#L25-L31

Reply:
Ok, I think you outlined a simple solution here, switching out 
`related=str(old.screening_status)` 
for 
`related=old.screening_status or '-'`